src/lib.rs

```diff
@@ -135,7 +135,7 @@
 //! be separate crates that pull in the dependencies they require.
 
 // Catch documentation errors caused by code changes.
-#![deny(intra_doc_link_resolution_failure)]
+#![deny(broken_intra_doc_links)]
```

Comment:
We cannot make this change until MSRV is at least 1.48.0 (see https://github.com/zcash/librustzcash/issues/279).